.github/workflows/continuous-integration.yml

```diff
@@ -9,19 +9,30 @@ jobs:
     strategy:
       matrix:
         python-version: [3.6, 3.7, 3.8]
+        mongodb-version: [5.0.2]
+        storage-version: [mem, db]
+      # max-parallel: 1
+      fail-fast: false
```

Comment:
Do we still need this? As I recall we added this as an attempt to resolve the GitHub actions timing out, but that ended up being resolved via a code change instead.